assets/src/edit-story/components/header/buttons/styles.js

```diff
@@ -25,10 +25,9 @@ import styled from 'styled-components';
 import { Icons } from '../../../../design-system';
 
 export const WarningIcon = styled(Icons.ExclamationTriangle)`
-  color: $(({theme}) => theme.DEPRECATED_THEME.colors.fg.white);
-  width: 14px;
-  height: 14px;
-  margin-left: 8px;
+  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.fg.white};
+  width: 32px !important;
```

Comment:
size is getting overridden in a shared button above this.

Reply:
I skipped this one as we're redoing the button completely in #6277 anyway, but thanks for updating!